extensions/tana/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Tana Changelog
 
+## [Fix Compatibility Issues] - 2023-12-01
+
+Fixed a 401 error when adding note
```

Comment:
Please add a note indicating an API token update is needed, referencing updated readme.